specutils/spectra/spectral_region.py

```diff
@@ -0,0 +1,152 @@
+import numpy as np
+import astropy.units as u
+
+
+class SpectralRegion:
+    """
+    A `SpectralRegion` is a container class enables some simplicty
+    in defining and passing a region (interval) for a spectrum.
+
+    In the future, there might be more functionality added in here and there
+    is some discussion that this might/could move to
+    `Astropy Regions <http://astropy-regions.readthedocs.io/en/latest/>`_.
```

Comment:
docstring for the ``__init__`` should go here rather than in ``__init__`` (and a corresponding convention is to put ``__init__`` at the top, before ``from_center`` so that "it's" docstring is as close as possible